Film Locations/Film Locations/MenuViewController.swift

```diff
@@ -62,17 +62,34 @@ class MenuViewController: UIViewController {
             
             // call before adding child view contoller's view as subview
             activeVC.didMove(toParentViewController: self)
-            activeViewController?.view.addSubview(toggleButton)
+            //activeViewController?.view.addSubview(toggleButton)
+            
+            setupToggleButton()
         }
     }
     
     private func setupToggleButton() {
-        toggleButton.frame = CGRect(x: 13, y: 18, width: 30, height: 30)
+        toggleButton.frame =  CGRect.zero
```

Comment:
We should not have hard coded frames in the code. So now to make it constraints, it has to be relative to active view controller as we are adding toggle button as subview. 
We have two options, 
1) We can set toggle button relative to MenuViewContoller's view and bring to front everytime we change active view controller 
2) Or add everytime. 
I chose second option as it was easier to change right now.